pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -712,6 +723,42 @@
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+ ::  +handle-cache-req: respond with cached value, 404 or 500
+  ++  handle-cache-req
+    |=  [authenticated=? =request:http =path]
+    |^  ^-  (quip move server-state)
+    ?.  authenticated
+      (error-response 403 ~)
+    ?.  =(%'GET' method.request)
+      (error-response 405 "may only GET scries")
+    =/  cache-entry  (~(get of cache.state) path)
+    ?~  cache-entry
+      (error-response 404 "no cache entry for that binding")
+    ?-    -.u.cache-entry
+        %payload
+      =/  payload  simple-payload.u.cache-entry
+      %-  handle-response
+      :*  %start
+          response-header.payload
+          data.payload
+          complete=%.y
+      ==
+        %scry
+      =/  res=(unit (unit cage))  (rof ~ %cx [our %base da+now] path.u.cache-entry)
```

Comment:
The `%scry` path should specify the `case` with a version number, as `da+now` scries cannot be cached.